archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII
+seq:
+  - id: files
+    type: file
+    repeat: eos
+types:
+  file:
+    seq:
+      - id: header
+        type: file_header
+      - id: path_name
+        type: str
+        size: header.path_name_size - 1
+      - id: string_terminator
+        contents: [0x00]
+      - id: path_name_padding
+        contents: [0x00]
+        if: header.path_name_size % 2 == 1
+      - id: file_data
+        size: header.file_size.least_significant_bits + header.file_size.most_significant_bits * 65536

```

Comment:
Thanks, agreed that's a good idea. I'll look into using value instances more in the future.